deeprank2/utils/parsing/__init__.py

```diff
@@ -65,7 +65,7 @@ def get_vanderwaals_parameters(self, atom: Atom) -> VanderwaalsParam:
                     type_ = action["TYPE"]
 
         if type_ is None:
-            _log.warning(f"Atom {atom} is unknown to the forcefield; vanderwaals_parameters set to (0.0, 0.0, 0.0, 0.0)")
+            _log.warning(f"Atom {atom} is unknown to OPLS, the forcefield used in deeprank2; vanderwaals_parameters set to (0.0, 0.0, 0.0, 0.0)")
```

Comment:
I think it would be nice if we could add some kind of information about what this means and/or how to proceed (or a link to such information, if the warning would otherwise become too long).
I think currently what would happen if a user gets this warning, unless they have a really high level of expertise, is that they are either scared off and and give up, or file a bug report/issue about this (or find the old one).

Also, maybe rather than sending the warning 100s of times (I think if it happens once in a file/dataset, it will happen a lot), it would be useful to count (or make a list of) all the times this happens and create a single warning for the entire set.

Reply:
I agree, I added a link to the docs/features.md section.

It would be nice to provide a more aggregate type of warning, but this would require to carefully edit the code and to add tests. Considering the constraints we have, I think we can stick to the verbose warnings. Not ideal, but they do the job.